### DISCUSSION TITLE
Why loop: true doesn't work for scrolling slider right

### DISCUSSION THREAD

**Discussion by itelniy** [nolimits4web/swiper#7086] [Q&A]
Hi! I have a swiper slider with 4 slides (as an example) and with loop: true and slidesPerView: 3 (in this case).
So I see that the loop feature doesn't work when I scroll the slider on the right side but works for the left-side scrolling, it works like this in cases:
- 3 slidesPerView, 4 slides
- 3 slidesPerView, 5 slides
- 3 slidesPerView, 7 slides
- 3 slidesPerView, 8 slides
- ...

BUT works for 3 slidesPerView and 6 slides

Code:
```
var testimonialsSlider = new Swiper('.js-testimonials-slider', {
            slidesPerView: 1,
            spaceBetween: 24,
            loop: true,
            navigation: {
                nextEl: ".testimonials-slider-holder .swiper-button-next",
                prevEl: ".testimonials-slider-holder .swiper-button-prev",
            },
            pagination: {
                el: ".testimonials-slider-holder .swiper-pagination",
                clickable: true,
            },
            breakpoints: {
                991: {
                    slidesPerView: slidesPerViewData,
                },
            }
        });
```
Please help me how to make loop work on both sides)

**Comment by james0r** [comment]
@itelniy Might wanna check out https://github.com/nolimits4web/swiper/issues/6472 . I think this bug appeared on version 9 after Swiper changed to more efficiently duplicate slides, but had this side effect. I think I dealt with this by passing slidesPerView based on how many items I counted in PHP.
